### PR TITLE
Mark api

### PR DESCRIPTION
**Pull request by MarkSelyukov**
Got the microservice to build the URL request for the external API

### CODE REVIEW

**Comment by MarkSelyukov** [comment]
Got the start done

**Comment by Grantlal** [comment]
Hey I'm ultimately really happy with this PR, if no one else looks it over we can merge it soon

**Comment by Grantlal** [comment]
Actually the more I'm looking at it, I just don't know what Retrieve data does. Obviously it retrieves data but with that type of name it makes it sound like it's service agnostic. When reading through the retrieve data file it looks like it's only for recipes? I would likely change the name of this file to something more specific if possible :)

File: src/main/java/Classes/RetrieveData.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    //Can make private and make a getter function
+    Double calories;
+    Double totalTime;
+
+    //Can make private and make a getter function
+    public List<Recipe> recipeList = new ArrayList<>();
+    HashMap<String, JSONArray> mapArray = new HashMap<>();
+    HashMap<String, JSONObject> mapObject = new HashMap<>();
+
+    public RetrieveData(String search, String ingr, String diet, String health, String cuisineType, String mealType, String dishType, String calories, String excluded) throws IOException, ParseException {
```
Comment (Grantlal) [inline src/main/java/Classes/RetrieveData.java]: What's the differrence between cuisine, meal, and dishtype?

**Comment by Grantlal** [comment]
Hey we need to return the image URL as well, and I'm having problems implementing that. Something we could pair on?

**Comment by MarkSelyukov** [comment]
We can pair or I can do it

________________________________

**Pull request merged**